docs/why-stacked.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: "Why Stacked"
+sidebar_position: 0
+---
+
+# What is Stacked
+
+Stacked is the production framework for Flutter. For teams or solo developers that require production level code, Stacked makes it easy to achieve. [FilledStacks](https://www.youtube.com/filledstacks) is a mobile software development agency with experience using Native, Xamarin and now Flutter to develop over 30 applications for large companies. We know what is required to build big, **scalable**, **testable** and **maintanable applications**, this is the core focus of Stacked.
+
+- **Scalable**: Stacked is built to make your team scalable and keep your productivity high. With strict principles and a strong opinion on how to develop functionality, you or your team will have a clear guide around adding or maintaining features.
+
+- **Testable**: We put an emphasis on unit tests and our architecture is desiged to make unit testing any part of your business logic or state as easy as possible.
+
+- **Maintainable**: With the strong opinions we have and strict principles to follow your code base can scale consistently and without worry of turning into spaghetti as you grow.
+
+## Why Stacked?
+
+Flutter is a framework that allows you to build your UI using code. This is amazing, but also dangerous. If there is not careful thought put into keeping these two separate, you will end up with UI files that are mixed with business logic and that proves almost impossible to unit test. Maintenance, testing and readabiliy of your code becomes more difficult which means over time you slow down. Stacked aims to do the opposite. Overtime, when using Stacked it actually makes you move faster. With our structure and guides you'll be able to build extremely large applications and feel like you're still working on a simple 2 screen application.
+
+:::info
+
+We work with many teams of developers, we're certain using our framework will benefit your code health, quality and maintenance in the long run.
```

Comment:
I don't like this line; here should be a 'quick entry' for example

```
Like what you hear? <Get Started> or head on over to our <API Reference> or collection of <examples> 
```

Reply:
Yeah, I was doubtful of this. I like your idea more.